rails/muffi_template/.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+cache: bundler
+bundler_args: "--without production"
+services:
+- postgresql
+sudo: false
+before_script:
+- bundle install
+- RAILS_ENV=development bundle exec rake db:create db:migrate
```

Comment:
Don't we need a script here?:
```yml
script:
  - bundle exec rubocop
  - bundle exec rspec
```

Or will travis magically figure it out?

Reply:
Good question. Definitely, it won't know about Rubocop :) Although, it ran my example tests.
Thanks for this, I'll add it.